product/views/item_view.py

```
@@ -144,3 +141,26 @@ def get(self, request, item_id):
+class ItemSellerContactView(APIView):
+    """
+    View to retrieve contact information of the seller of a specific item.
+    """
+    permission_classes = [IsAuthenticated]
+    authentication_classes = [CookieJWTAuthentication]
+
+    def get(self, request, item_id):
+        try:
+            item = Item.objects.get(id=item_id)
+        except Item.DoesNotExist:
+            raise ItemNotFoundException()
```

Ruff (0.8.2)

Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)